### PR TITLE
[feat] Scale figure size based on the box size

### PR DESCRIPTION
**Pull request by roubkar**
- Apply scale to the figure container based on the box size ratio
- Tune box styles

### CODE REVIEW

**Comment by roubkar** [comment]
Resolves https://github.com/aimhubio/aim/issues/1730

File: aim/web/ui/src/modules/BaseExplorer/components/BoxFullViewPopover/BoxFullViewPopover.tsx [hunk 1]
```diff
@@ -181,34 +181,36 @@ function BoxFullViewPopover({
                           {groupConfigKey}
                         </Text>
                         <Text tint={70}>
-                          ({data.groups[groupConfigKey].items_count_in_group}
+                          ({data.groups[groupConfigKey].items_count_in_group}{' '}
```
Comment (VkoHov) [inline aim/web/ui/src/modules/BaseExplorer/components/BoxFullViewPopover/BoxFullViewPopover.tsx:184]: let's replace it to `({data.groups[groupConfigKey].items_count_in_group} items in this group)`

File: aim/web/ui/src/modules/BaseExplorer/components/BoxVirtualizer/BoxVirtualizer.scss [hunk 2]
```diff
@@ -14,8 +14,8 @@
   width: 200px;
   height: 30px;
   background-color: #fff;
-  border-bottom: $border-separator;
-  border-right: $border-separator;
+  border-bottom: 1px solid $pico-30;
```
Comment (VkoHov) [inline aim/web/ui/src/modules/BaseExplorer/components/BoxVirtualizer/BoxVirtualizer.scss:17]: Can we have a variable for that?

File: aim/web/ui/src/modules/BaseExplorer/index.tsx [hunk 3]
```diff
@@ -75,7 +75,7 @@ function createExplorer(config: IExplorerConfig): () => React.ReactElement {
   };
 
   const engine = createEngine(engineConfig);
-
+  console.log(engine);
```
Comment (VkoHov) [inline aim/web/ui/src/modules/BaseExplorer/index.tsx:78]: Remove the console.log statement.

**Review by rubenaprikyan — APPROVED** [review]
Lgtm!

File: aim/web/ui/src/modules/BaseExplorer/components/BoxFullViewPopover/BoxFullViewPopover.tsx (hunk 1 above)
Comment (roubkar) [reply aim/web/ui/src/modules/BaseExplorer/components/BoxFullViewPopover/BoxFullViewPopover.tsx:184]: It is written in that way @VkoHov, just formatted by `prettier`

**Pull request merged**